analysis/generate_report.py

```diff
@@ -65,6 +65,15 @@ def get_arg_parser():
         type=int,
         help=('The last time (in seconds) during an experiment to include in '
               'the report.'))
+
+    parser.add_argument(
+        '-x',
+        '--max-num-trials',
+        default=None,
+        type=int,
+        help=('The maximum number of trials per fuzzer-benchmark to include in '
+              'the report. Useful if you want to join experiments that used '
+              'different numbers of trials.'))
```

Comment:
Consider adding something discouraging here, e.g.: "The maximum number of trials per fuzzer-benchmark to include in the report. Using this options drops measurement data, lowers the precision of the report, and might lead to less statistically significant results. Only use this feature if you know what you are doing".